tests/Codge.Generator.Test/TestStore/XsdModel/Model.xsd

```diff
@@ -132,6 +132,19 @@
     </xs:complexContent>
   </xs:complexType>
 
+  <xs:complexType name="complexTypeWithInlineTypeLessElement">
+    <xs:sequence>
+      <xs:element name="TypeLess">
```

Comment:
WithInlinedType?

Reply:
Actually, there is already ElementWithEmbededTypeInPlace, do you need another one?

Reply:
I do, only because that one has an inline optional enumeration which is an extension of a base type (string in that case).

This new one is an inline complex type within a complex type